Makefile

```diff
@@ -54,6 +64,25 @@ pb_generate:
 		--grpc_python_out=./proto ./proto/workload.proto
 
 
+#------------------------------------------------------------------------
+# Internal targets
+#------------------------------------------------------------------------
+
+# Targets that aren't normally manually run, so not in the generated help
+
+black:
+	@echo "Running black (check only)."
+	$(PIPENV_CMD) run black --check --diff --color src test
+
+flake8:
+	@echo "Running flake8."
+	$(PIPENV_CMD) run flake8 --statistics --doctests --ignore E203,E266,E501,W503 src test
+
+mypy:
+	@echo "Running mypy."
+	$(PIPENV_CMD) run mypy -m src
+
+
```

Comment:
:100: for adding these tasks as _internal_

For those not familiarized with the `flake8` options and rules, here are some links.
Rules: https://www.flake8rules.com/
Options: 
```
--statistics          Count errors and warnings.
--doctests            check syntax of the doctests
```
[Source](https://flake8.pycqa.org/en/latest/manpage.html#options)